playbooks/maas-openstack-octavia.yml

```diff
@@ -67,6 +67,12 @@
   hosts: octavia-api
   gather_facts: false
   tasks:
+    - name: Add load-balancer member role
+      shell: ". /root/openrc && openstack role add --project $OS_PROJECT_NAME --user $OS_USERNAME load-balancer_member"
```

Comment:
We should consider the read-only roles for maas: role:load-balancer_observer or role:load-balancer_global_observer

https://docs.openstack.org/octavia/latest/configuration/policy.html